tests/sparseml/transformers/test_transformers.py

```diff
@@ -108,14 +133,15 @@ def setup(self, model_stub, recipe_present, task):
     def test_load_weights_apply_recipe(self, setup):
         model, recipe_present, task = setup
         model_path = model.framework_files[0].dir_path
+        _attempt_copy_original_recipe(model_path)
```

Comment:
I'd rather execute this function on the setup. Should be more foolproof once we keep adding more tests / decide to run tests selectively.